edge-modules/ModuleRestarter/docker/linux/arm32v7/base/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG base_tag=2.1.10-bionic-arm32v7
```

Comment:
Remove this file; this Linux ARM32 base image is same across all the modules; which you can find the docker file under  edge-modules\SimulatedTemperatureSensor\docker\linux\arm32v7\base folder.